src/test/resources/log4j.xml

```diff
@@ -50,6 +50,8 @@
     <appender-ref ref="file" />
   </root>
 
-  <loggerFactory class="org.owasp.esapi.reference.Log4JLoggerFactory"/>
+  <!-- 
+  <loggerFactory class="org.owasp.esapi.logging.log4j.Log4JLoggerFactory"/>
+ -->
```

Comment:
This is the bit that loads the SPI backdoor to Log4J.  This is why we will see logs for org.apache.commons in the test logs file.  (really confused me that the Log4JLogFactory was getting loaded while I was testing the JavaLogFactory)

This should be uncommented.